database/migrations/2023_10_30_034209_create_activity_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('activity', function (Blueprint $table) {
+            $table->id('activity_id');
+            $table->foreignId('activity_type_id')->constrained(
+                table: 'activity_type', column: 'activity_type_id'
+            );
+            $table->year('academic_year');
+            $table->integer('term');
+            $table->date('start_date');
+            $table->time('start_time');
+            $table->date('end_date');
+            $table->time('end_time');
```

Comment:
Actually, I just realized maybe we can update the ERD regarding this part.
MySQL has type for 'DATETIME' which is basically combination of `DATE` and `TIME`.

With that, we will only need a column for `start` and `end`.

It's up to you though.

Reply:
Yeah, I agree, that's probably how the frontend would work with it. I remember I separated it in the erd because it was separated in the crs document. If ever, let's just process it in the api if they want it separate.

Reply:
I'll also update the start date and end date in class table to use datetime